drivers/gpio/gpio_andes_atcgpio100.c

```diff
@@ -49,30 +42,22 @@
 #define INT_POSITIVE_EDGE       0x6
 #define INT_DUAL_EDGE           0x7
 
-#define PULL_CONFIGURED         BIT(31)
-#define DEBOUNCE_CONFIGURED     BIT(29)
 #define DF_DEBOUNCED_SETTING    (0x80000003)
 
 #define GPIO_BASE(dev) \
 	((const struct gpio_atcgpio100_config * const)(dev)->config)->base
 
-#define GPIO_CFG(dev)  (GPIO_BASE(dev) + REG_CFG)
 #define GPIO_DIR(dev)  (GPIO_BASE(dev) + REG_DIR)
 #define GPIO_DIN(dev)  (GPIO_BASE(dev) + REG_DIN)
 #define GPIO_DOUT(dev) (GPIO_BASE(dev) + REG_DOUT)
 #define GPIO_DCLR(dev) (GPIO_BASE(dev) + REG_DCLR)
 #define GPIO_DSET(dev) (GPIO_BASE(dev) + REG_DSET)
-#define GPIO_PUEN(dev) (GPIO_BASE(dev) + REG_PUEN)
-#define GPIO_PTYP(dev) (GPIO_BASE(dev) + REG_PTYP)
 #define GPIO_INTE(dev) (GPIO_BASE(dev) + REG_INTE)
 #define GPIO_IMD(dev, idx) (GPIO_BASE(dev) + REG_IMD0 + (idx * 4))
 #define GPIO_ISTA(dev) (GPIO_BASE(dev) + REG_ISTA)
 #define GPIO_DEBE(dev) (GPIO_BASE(dev) + REG_DEBE)
 #define GPIO_DEBC(dev) (GPIO_BASE(dev) + REG_DEBC)
 
-#define INWORD(x)     sys_read32(x)
```

Comment:
Removing `INWORD()` and `OUTWORD()` definitely makes the code clearer.

Thanks!

Is it possible to have this change split into two commits? Perhaps the first commit taking care of the `INWORD()` / `OUTWORD()` changes and the second commit adding `gpio_atcgpio100_port_get_dir()`

Reply:
Yes, I'll split the changes into two commits.